packages/react-aria-components/src/TagGroup.tsx

```diff
@@ -213,8 +213,10 @@ function TagItem({item}) {
   let {rowProps, gridCellProps, removeButtonProps, ...states} = useTag({item}, state, ref);
 
   let {hoverProps, isHovered} = useHover({
-    isDisabled: !states.allowsSelection
+    isDisabled: state.disabledKeys.has(item.key)
   });
+  // TODO: Do we consider a tag with a link or with remove enabled as "interactive"?
```

Comment:
open question

Reply:
I would say yes. A Tag is interactive unless it is disabled.